Package:     capi4hylafax
Subject:     optimize config paths
Author:      Lionel Elie Mamane <lmamane at debian.org>
Forwarded:   no
Last-Update: 2012-06-12

Update some configuration defaults, escpecially change some paths
for security reasons.
---

diff -urN s01/config.faxCAPI s02/config.faxCAPI
--- s01/config.faxCAPI	2012-06-12 19:45:18.812012374 +0200
+++ s02/config.faxCAPI	2012-06-12 19:54:24.729634185 +0200
@@ -39,7 +39,7 @@
 #
 # default   /var/spool/fax
 #
-SpoolDir:               /var/spool/fax
+SpoolDir:               /var/spool/hylafax
 
 
 # "FaxRcvdCmd" gibt das Skript an, das ausgeführt wird, sobald ein Fax
@@ -52,7 +52,7 @@
 #
 # default   /var/spool/hylafax/bin/faxrcvd
 #
-FaxRcvdCmd:             /var/spool/fax/bin/faxrcvd
+FaxRcvdCmd:             /var/spool/hylafax/bin/faxrcvd
 
 
 # "PollRcvdCmd" gibt das Skript an, das ausgeführt wird, sobald ein
@@ -67,7 +67,7 @@
 #
 # default   /var/spool/hylafax/bin/pollrcvd
 #
-PollRcvdCmd:             /var/spool/fax/bin/pollrcvd
+PollRcvdCmd:             /var/spool/hylafax/bin/pollrcvd
 
 
 # "FaxReceiveUser" und "FaxReceiveGroup" geben den User und die Gruppenkennung
@@ -87,7 +87,7 @@
 #
 # default   ""
 #
-FaxReceiveUser:         fax
+FaxReceiveUser:         uucp
 FaxReceiveGroup:        dialout
 
 
@@ -97,7 +97,7 @@
 #
 # default   "/tmp/capifax.log"
 #
-LogFile:                /tmp/capifax.log
+LogFile:                /var/spool/hylafax/log/capi4hylafax
 
 
 # Der Befehl "LogTraceLevel" gibt die Menge der Daten an, die in ein LogFile
@@ -319,7 +319,7 @@
         #
         # default   0
         #
-        AcceptSpeech:           0
+        AcceptSpeech:           1
 
 
         # Sofern man DDI nutzen will, sollte man "UseDDI" auf 1 stellen.
diff -urN s01/setupconffile s02/setupconffile
--- s01/setupconffile	2006-08-15 11:13:25.000000000 +0200
+++ s02/setupconffile	2012-06-12 19:54:24.733644042 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 #   default values
 #
@@ -17,7 +17,7 @@
 DDISamples="$DDISamples\nSample for DDI Length   : 3"
 DDISamples="$DDISamples\nSample for Incoming DDIs: 10, 100-300"
 
-ConfigFileName="/etc/config.faxCAPI"
+ConfigFileName="/etc/hylafax/config.faxCAPI"
 if [ ! -e $ConfigFileName ] ; then
     echo "setupconffile: Can't find CAPI4HylaFax config file ($ConfigFileName) !"
     return 1
@@ -274,3 +274,4 @@
 rem_res
 return 0
 }
+main_config_dialog
diff -urN s01/src/defaults.h.in s02/src/defaults.h.in
--- s01/src/defaults.h.in	2006-08-15 11:13:33.000000000 +0200
+++ s02/src/defaults.h.in	2012-06-12 19:54:24.737653902 +0200
@@ -30,12 +30,12 @@
 #define HYLAFAX_RECV_CMD            "bin/faxrcvd"
 #define HYLAFAX_POLL_CMD            "bin/pollrcvd"
 #define HYLAFAX_SEQF_FILE           "seqf"
-#define HYLAFAX_XFERLOG             "etc/xferfaxlog"
+#define HYLAFAX_XFERLOG             "log/xferfaxlog"
 
 #define DEFAULT_FAX_USER            "uucp"
 #define DEFAULT_DEVICE_NAME         "faxCAPI"
 #define DEFAULT_SPOOL_DIRECTORY     "@hylafaxspooldir@"
-#define DEFAULT_CONFIG_FILE         "/etc/config.faxCAPI"
+#define DEFAULT_CONFIG_FILE         "/var/spool/hylafax/etc/config.faxCAPI"
 #define DEFAULT_FILE_MODE           0600
 #define DEFAULT_RECVPROG_NAME       "CapiFaxRecv"
 #define DEFAULT_SENDPROG_NAME       "CapiFaxSend"
